.github/workflows/upstream-nightly.yml

```diff
@@ -22,19 +22,14 @@ on:
 
 jobs:
   upstream-dev:
-    runs-on: ubuntu-20.04-16core
+    runs-on: ubuntu-latest
```

Comment:
What's the motivation for changing the runner type here? I don't know the details, but I thought that the default runners only had 2 cores - would this affect the runtimes?

Reply:
Yes, it approximately doubles the runtime, from [9 minutes](https://github.com/jax-ml/jax/actions/runs/13009945085) to [18 minutes](https://github.com/jax-ml/jax/actions/runs/12998453238)

I suspect this originally was used becuase it was used in the workflow that this structure was copied from; 16 core machines are more expensive than standard machines, and I wasn't sure that it was needed. I think given that this is a nightly workflow, 18 minutes is fine – what do you think?

Reply:
That sounds totally reasonable to me!